modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -170,6 +170,10 @@ inline int toCV(ONNXTensorElementDataType prec) {
     case ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT8: return CV_8U;
     case ONNX_TENSOR_ELEMENT_DATA_TYPE_FLOAT: return CV_32F;
     case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT32: return CV_32S;
+    case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64: {
```

Comment:
As option ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64 can be mapped CV_32S and setup the flag is_conversion
to prevent handling -1 in every place.

What do you think ? Let's ask someone else for example

Reply:
Or return `tuple<type, bool>`

Reply:
Return tuple. toCV function should receive tensorInfo parameter for flag .